scribejava-apis/src/test/java/com/github/scribejava/apis/examples/OdnoklassnikiExample.java

```diff
@@ -18,14 +18,13 @@
 
     public static void main(String... args) {
         // Replace these with your client id and secret
-        final String clientId = "your client id";
-        final String publicKey = "your api secret";
-        final String clientSecret = "your client secret";
+        final String clientId  = "your api client id";
+        final String publicKey = "your api public key";
```

Comment:
Yes, but some developers in first time can't understand what kind of key we want to receive.
Explicit name can resolve this mistake.